### ISSUE TITLE
Cannot use more than one controller

### ISSUE THREAD

**Issue by Jeremiidesu** [ppy/osu-framework#3518]
When connecting two or more controllers, only the first one will respond to inputs.

**Comment by smoogipoo** [comment]
Joystick or midi controller?

**Comment by Jeremiidesu** [comment]
Joysticks, not MIDI controllers ; updated the title and first comment to reflect this

**Comment by Jeremiidesu** [comment]
Additional stuff I just noticed: 

- The joystick test scene only handles the first connected joystick
- A game project of mine handles input for all connected joysticks as the same joystick.

In any case, not what could be expected from proper multiple joystick support

**Comment by peppy** [comment]
Just to confirm, all your controllers are correctly firing events?

**Comment by Jeremiidesu** [comment]
Yes, they all do fire events, with no way to determine the controller that fired the event.
